docs/getting-started/integrations-cdn/laravel.md

```diff
@@ -49,80 +46,121 @@ Once the project has been prepared, create an `assets/vendor/ckeditor5.js` file
 └── ...
 ```
 
-Inside the file, paste the JavaScript code from CKEditor&nbsp;5 Builder. The code will differ depending on your chosen preset and features. But it should look similar to this:
+First, modify the `welcome.blade.php` file in the `resources/views` directory to include the CKEditor&nbsp;5 scripts and styles. All necessary scripts and links are in the HTML snippet below. You can copy and paste them into your template. Open-source and premium features are in separate files, so there are different tags for both types of plugins. Add tags for premium features only if you use them.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>CKEditor 5 - Quick start CDN</title>
+	<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+	<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.umd.js"></script>
+	<!-- Add if you use premium features. -->
+	<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5-premium-features/{@var ckeditor5-version}/ckeditor5-premium-features.css" />
+	<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5-premium-features.umd.js"></script>
+	<!--  -->
+	<style>
+	.main-container {
+		width: 795px;
+		margin-left: auto;
+		margin-right: auto;
+	}
+    </style>
+</head>
+<body>
+	<div class="main-container">
+		<div id="editor">
+			<p>Hello from CKEditor 5!</p>
+		</div>
+    </div>
+</body>
+</html>
+```
+
+Both previously attached scripts expose global variables named `CKEDITOR` and `CKEDITOR_PREMIUM_FEATURES`. You can use them to access the editor class and plugins. In our example, we use object destructuring (JavaScript feature) to access the editor class from the open-source global variable with a basic set of plugins. You can access premium plugins from the other variable the same way. Then, we pass the whole configuration to the `create()` method. Be aware that you need a proper {@link getting-started/licensing/license-key-and-activation license key} to use premium features.
 
 ```js
```

Comment:
From the flow of text, it's a bit confusing where this code block should appear, create some placeholder in the index.html?

Reply:
I would start showing it in the <script> from start I think.

Reply:
I wanted to show the JavaScript snippet, explain it a bit, and then show it in the template. Actually, it's a good idea.